tools/chplvis/InfoBar.cxx

```diff
@@ -34,6 +33,20 @@ static const int Y_OFF = 10;
 static const int Y_OFF = 0;
 #endif
 
+// Local utility functions
+
+// Computation of color for use in displays
+
+Fl_Color heatColor ( double val, double max ) {
```

Comment:
small nitpick: A few function definitions / declarations vary stylistically from the rest of the file.

e.g., I would expect:

    Fl_Color heatColor(double val, double max) {